src/index.html

```diff
@@ -5,10 +5,91 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>John Roberts' CV</title>
-    <link rel="stylesheet" href="./css/index.scss" />
+    <link rel="stylesheet" href="/css.81192ade.css" />
+    <script src="/css.81192ade.js"></script>
   </head>
+
   <body>
-    <div id="main"></div>
-    <script src="./js/index.js"></script>
+    <script src="/js.00a46daa.js"></script>
+    <nav>
+      <a href="/index.html">HOME</a>
+      <button class="btn">DOWNLOAD MY FULL RESUME</button>
+    </nav>
+
+    <div id="main">
+      <!-- <h1 style="color: white">John Roberts</h1> -->
+      <h1 style="color: black">John Roberts</h1>
+      <h5 style="color:#938582">BUSINESS EXPERT</h5>
+    </div>
+
+    <!--Rollbar-->
+    <div class="rollbar">
+      <h3>ABOUT ME</h3>
+      <button class="btn">DOWNLOAD MY FULL RESUME</button>
+      <div class="aboutme">
+        <p id="am1">
+          I'm a paragraph. Click here to add your own text and edit me. It’s
+          easy. Just click “Edit Text” or double click me to add your own
+          content and make changes to the font. Feel free to drag and drop me
+          anywhere you like on your page. I’m a great place for you to tell a
+          story and let your users know a little more about you.
+        </p>
+      </div>
+      <div class="aboutme">
+        <p id="am2">
+          I'm a paragraph. Click here to add your own text and edit me. It’s
+          easy. Just click “Edit Text” or double click me to add your own
+          content and make changes to the font. Feel free to drag and drop me
+          anywhere you like on your page. I’m a great place for you to tell a
+          story and let your users know a little more about you.
+        </p>
+      </div>
+    </div>
+
+    <!--SKILLS-->
+    <div class="skills">
+      <h3>SKILLS</h3>
+      <div>
+        <!--Skills blocks-->
+      </div>
+    </div>
+
+    <!--QUICK ID-->
+    <div class="quick_id">
+      <h3>QUICK ID</h3>
+      <table id="tbquick_id">
```

Comment:
Good to keep using the same name conventions (snake_case) for the tag attributes 👍